source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -0,0 +1,208 @@
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_000.png
+
+2. A configuration wizard for adding a new relying party trust opens. In the **Welcome** screen, choose **Claims aware** and click **Start**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_001.png
+
+3. In the **Select Data Source** screen, select the option **Enter data about the relying party manually**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_002.png
+
+4. In the **Specify Display Name** screen, enter a **Display Name** to recognize the trust, such as ``Mattermost``, and add any notes you want to make.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_003.png
+
+5. In the **Configure Certificate** screen, leave the certificate settings at their default values.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_004.png
+
+However, if you would like to set up encryption for your SAML connection, click the **Browse** button and upload your Service Provider Public Certificate.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_005.png
+
+6. In the **Configure URL** screen, select the option **Enable Support for the SAML 2.0 WebSSO protocol** and enter the **SAML 2.0 SSO service URL**, which is of the form ``https://<your-mattermost-url>/login/sso/saml`` where ``https://<your-mattermost-url>`` should typically match the `Mattermost Site URL <https://docs.mattermost.com/administration/config-settings.html#site-url>`__.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_006.png
+
+7. In the **Configure Identifiers** screen, enter the **Relying party trust identifier** (also known as the **Identity Provider Issuer URL**) of the form ``https://<your-idp-url>/adfs/services/trust`` and click **Add**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_007.png
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_008.png
+
+Add your **SAML 2.0 SSO service URL** from above the same way.
+
+8. In the **Choose Access Control Policy** screen select the access control policy suitable for your environment. This guide assumes the default values **Permit everyone** and an unchecked box are kept.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_009.png
+
+9. In the **Ready to Add Trust** screen, you can review your settings.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_010.png
+
+10. In the **Finish** screen, select the option **Configure claims issuance policy for this application**, and click **Close**. 
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_011.png
+
+	You will now exit the configuration wizard and a **Claim Rules** editor opens.
```

Comment:
I think this can be removed as the previous step indicated the wizard would be closed and the next section refers to the Claim Rules creation process.

Reply:
agreed and removed